gen/definitions/bgp_peer_template_address_family.yaml

```diff
@@ -1,22 +1,17 @@
 ---
 name: BGP Peer Template Address Family
 class_name: bgpPeerAf
-dn: sys/bgp/inst/dom-[%s]/peercont-[%s]/af-[%s]
+dn: sys/bgp/inst/dom-[default]/peercont-[%s]/af-[%s]
 ds_description: This data source can read the BGP peer template address family configuration.
 res_description: This resource can manage the BGP peer template address family configuration.
 doc_path: Routing%20and%20Forwarding/bgp:PeerAf/
 parents:
   - BGP Peer Template
 children:
   - BGP Peer Template Max Prefix
+references:
+  - BGP Peer Address Family
```

Comment:
Aren't we referring to the "BGP Address Family" here?

Reply:
Fixed references.